stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -11,7 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 import Swift
-import _Concurrency
+@_spi(ConcurrencyExecutors) import _Concurrency
```

Comment:
TODO: cant make use of it after all because we need the executor initializer from an inlinable function.